Tutorials/Creating-and-distributing-a-package/index.md

```diff
@@ -0,0 +1,500 @@
+# Creating and distributing a package
+
+This tutorial will take you through creating a brand new package in Umbraco, distributing it on the package repository on Our Umbraco, and finally automating the update flow and including it in CI/CD.
+
+## Agenda
+
+- Prerequisites
+- Creating a testsite locally
+- Creating a package from the backoffice
+- Creating a draft package on Our
+- Pushing your package to Github
+- Pack up your package locally using UmbPack 
+- Pushing your package to Our using UmbPack
+- Deploy your package using Github Actions
+- Archive older versions on push
+
+## Prerequisites
+
+To run this tutorial you will need the following:
+
+- Be able to run an Umbraco site locally
+- Git + Github account
+- Our member account with access to upload packages
+- Umbpack installed
```

Comment:
I wasn't sure whether this should always be UmbPack as you've used in some places or here where you have Umbpack ?

Reply:
```suggestion
- UmbPack installed
```
Understandable.. Naming is hard, and this name is complicated sometimes 😀 Let's make it conform to the other instances for now.